src/libs/telemetry/markOpenReportEnd.ts

```diff
@@ -1,11 +1,13 @@
 import Timing from '@libs/actions/Timing';
 import Performance from '@libs/Performance';
 import CONST from '@src/CONST';
+import {endSpan} from './activeSpans';
 
 /**
  * Mark all 'open_report*' performance events as finished using both Performance (local) and Timing (remote) tracking.
  */
-function markOpenReportEnd() {
+function markOpenReportEnd(reportId: string) {
+    endSpan(`${CONST.TELEMETRY.SPAN_OPEN_REPORT}_${reportId}`);
```

Comment:
This accounts only for this main event, so doesn't allow us to parametrize the origin of this action (which UI part triggered navigation). Do we want to cover that later?

Reply:
I think this should also not include `_<id>`, am I wrong?